Tools/jit/_writer.py

```diff
@@ -19,14 +19,14 @@ def _dump_footer(
     yield "        const _PyUOpInstruction *instruction, jit_state *state);"
     yield "    size_t code_size;"
     yield "    size_t data_size;"
-    yield "    symbol_mask trampoline_mask;"
+    yield "    symbol_mask shim_mask;"
```

Comment:
This member is indeed referring to the "little" trampolines:

```suggestion
    yield "    symbol_mask trampoline_mask;"
```

Reply:
Ahh, you're right. Thanks!